csgo_internal/csgo_internal/IEngineClient.h

```diff
@@ -137,7 +137,11 @@ class IEngineClient
 	virtual int                   GetAppID() = 0;
 	virtual Vector                GetLightForPointFast( const Vector &pos, bool bClamp ) = 0;
 	virtual void                  ClientCmd_Unrestricted1( char  const*, int, bool );
-	virtual void                  ClientCmd_Unrestricted( const char *szCmdString ) = 0; // 114
+	virtual void                  ClientCmd_Unrestricted( const char *szCmdString, const char *newFlag) = 0; // 114, new flag, quick testing shows setting 0 seems to work, haven't looked into it.
+	//Forgot to add this line, but make sure to format all unrestricted calls now with an extra , 0
+	//Ex:
+	//	I::Engine->ClientCmd_Unrestricted( charenc( "cl_mouseenable 1" ) , 0);
+	//	I::Engine->ClientCmd_Unrestricted( charenc( "crosshair 1" ) , 0);
```

Comment:
You could also leave the definition unmodified, and instead add this to IEngineClient.cpp

```c++
void IEngineClient::ClientCmd_Unrestricted(const char *szCmdString)
{
	typedef void(__thiscall* OriginalFn)(PVOID, const char*, char);
	return U::GetVFunc<OriginalFn>(this, 114)(this, szCmdString, 0);
}
```

Poached from UC: https://www.unknowncheats.me/forum/1515371-post5207.html